config/serverless.es.yml

```diff
@@ -12,6 +12,22 @@ xpack.serverless.observability.enabled: false
 enterpriseSearch.enabled: false
 xpack.fleet.enabled: false
 xpack.observabilityAIAssistant.enabled: false
+xpack.osquery.enabled: false
+
+## Fine-tune the search solution feature privileges. Also, refer to `serverless.yml` for the project-agnostic overrides.
+xpack.features.overrides:
+  ### Dashboards feature is moved from Analytics category to the Search one.
+  dashboard.category: "enterpriseSearch"
```

Comment:
nit: this reminds me that we need to change the category id from `enterpriseSearch` to `search` at some point.